### ISSUE TITLE
Document Migration from ftw

### ISSUE THREAD

**Issue by dextermallo** [coreruleset/go-ftw#166]
Hi go-ftw Team,

I am working on updating some documentation, and I found that some of the docs are missing in the go-ftw. (e.g., the format of the YAML test file)

And I found it in the ftw repo: https://github.com/coreruleset/ftw/tree/main/docs 

Should we migrate the documentation to go-ftw?

**Comment by theseion** [comment]
We're working on generating the documentation for the test file format. If we can get that off the ground soon then we don't have to. @fzipi?

**Comment by fzipi** [comment]
Exactly. I'm working on having this in the code. I'll update everything accordingly.

**Comment by fzipi** [comment]
@dextermallo The new repo with generated documentation is here: https://github.com/coreruleset/ftw-tests-schema

**Comment by fzipi** [comment]
@dextermallo Do you still have time to update this documentation?

**Comment by dextermallo** [comment]
Hi @fzipi, sure! Lemme do a quick check on this issue :)

**Comment by dextermallo** [comment]
Hi, I have another idea. Instead of making it under an individual repository, would it be better to make the official documentation as an all-in-one collection to ease the difficulty of referencing/searching for documentation?

The schema/usage for testing may be reused across different repositories. Putting it in the main documentation can make things easier.

Can also be put together in this issue: https://github.com/coreruleset/documentation/issues/133
